x/emissions/keeper/inference_synthesis/forecast_implied_inferences.go

```diff
@@ -128,16 +128,11 @@ func CalcForecastImpliedInferences(
 				// `j` is the inferer id. The nomenclature of `j` comes from the corresponding regret formulas in the litepaper
 				for _, j := range sortedInferersInForecast {
 					// Calculate the approximate forecast regret of the network inference
-					networkLossPerValue, err := networkCombinedLoss.Quo(forecastElementsByInferer[j].Value)
+					R_ik[j], err = networkCombinedLoss.Quo(forecastElementsByInferer[j].Value)
```

Comment:
these values are already log'd so should be a difference, not a quotient. recall: `log(a/b) = loga - logb`, and we're given `loga,logb` now instead of `a,b`.

This is probs why the unit test values didn't change by as much as i expected.

Reply:
Yes, You are correct....